build/configs/imxrt1020-evk/loadable_elf_apps/defconfig

```diff
@@ -1341,3 +1341,4 @@ CONFIG_IOTBUS_ADC=y
 # Security Framework
 #
 # CONFIG_SECURITY_API is not set
+CONFIG_PROD_HEADER="prod/fac.h"
```

Comment:
Please add newline at end of files.

Reply:
it's temporary fix. so I'll fix it after sync is done

Reply:
I don't know why you need a temporary fix. In this PR, you can make a perfect fix and then no need to modify again.